.github/workflows/security-test-workflow.yml

```diff
@@ -36,29 +36,27 @@ jobs:
         run: ./gradlew assemble
       - name: Pull and Run Docker
         run: |
-          plugin=`ls alerting/build/distributions/*.zip`
+          plugin=`basename $(ls alerting/build/distributions/*.zip)`
           list_of_files=`ls`
           list_of_all_files=`ls alerting/build/distributions/`
           version=`echo $plugin|awk -F- '{print $3}'| cut -d. -f 1-3`
           plugin_version=`echo $plugin|awk -F- '{print $3}'| cut -d. -f 1-4`
           qualifier=`echo $plugin|awk -F- '{print $4}'| cut -d. -f 1-1`
           candidate_version=`echo $plugin|awk -F- '{print $5}'| cut -d. -f 1-1`
-          docker_version=$version-$qualifier-candidate_version
+          docker_version=$version-$qualifier
 
-          if [ -z $candidate_version ]; then
-              candidate_version=$qualifier && qualifier=""
-              docker_version=$version
-          fi
+          [[ -z $candidate_version ]] && candidate_version=$qualifier && qualifier=""
```

Comment:
Is this setting `candidate_version` to `qualifier`'s value when it's empty? I don't see us using `$candidate_version` anymore below since `$plugin` is used to fetch the plugin zip, can it be removed?

Reply:
I would still keep it here just in case we change the strategy later to include candidate in docker image.
This is also good for us to track all these vars and its values.
No harm.

Reply:
Typically we should remove unused code and use the git history to reintroduce it if we need it. Not a blocker though, I'll approve.